src/cmake/ie_parallel.cmake

```diff
@@ -141,6 +141,16 @@ macro(ov_find_package_tbb)
                     list(APPEND TBB_IMPORTED_TARGETS ${target})
                 endif()
             endforeach()
+
```

Comment:
we can also add a check:
```cmake
if(TARGET TBB::tbbbind_2_5)
 ... all code with HWLOC
endif()
```
because without TBBBind, we don't have HWLOC

Reply:
Thanks, done!